src/AddressRegistry.Api.BackOffice.Abstractions/Requests/CorrectRegularizedAddressBackOfficeRequest.cs

```diff
@@ -0,0 +1,16 @@
+namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+
+    [DataContract(Name = "CorrigerenRegularisatieAdres", Namespace = "")]
+    public class CorrectRegularizedAddressBackOfficeRequest
```

Comment:
Is this the correct naming convention? See
https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry/Building/Commands/CorrectBuildingRealization.cs
https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry/Building/Commands/CorrectBuildingUnitRetirement.cs